bucket/fastfetch.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.8.2",
+    "description": "A neofetch-like tool for fetching system information and displaying them in a pretty way, written in pure C",
```

Comment:
> written in pure C

This was only true before Windows ( and macOS ) support was introduced, because [Win32 COM headers ( for WMI queries ) require C++ compilers](https://github.com/LinusDierheimer/fastfetch/blob/dev/src/util/windows/wmi.hpp).

Reply:
Please update the readme as well